en/README.md

```diff
@@ -0,0 +1,1138 @@
+    |-------------putEvents(Response)------------->|
+    |                                              |
+    |<-----------getLastInfo(Request)--------------|
+    |------------getLastInfo(Response)------------>|
+    |<------------putHostParents(Request)----------|
+    |-------------putHostParents(Response)-------->|
+    |                                              |
+    |<------------putArmInfo(Request)--------------|
+    |-------------putArmInfo(Response)------------>|
+    |                                          |   |
+    |                             Polling interval |
+    |                                          |   |
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    |                                              |
+    |-----------fetchItems(Request)--------------->|
+    |<----------fetchItems(Response)---------------|
+    |<------------putItems(Request)----------------|
+    |-------------putItems(Response)-------------->|
+    |                                              |
+    |-----------fetchHistory(Request)------------->|
+    |<----------fetchHistory(Response)-------------|
+    |<------------putHistory(Request)--------------|
+    |-------------putHistory(Response)------------>|
+    |                                              |
+    |-----------fetchTriggers(Request)------------>|
+    |<----------fetchTriggers(Response)------------|
+    |<-----------putTriggers(Request)--------------|
+    |                Use "ALL"option               |
+    |-----------putTriggers(Response)------------->|
+    |                                              |
+    |-------------fetchEvents(Request)------------>|
+    |<------------fetchEvents(Response)------------|
+    |<------------putEvents(Request)---------------|
+    |-------------putEvents(Response)------------->|
+    |                                              |
+    |--updateMonitoringServerInfo(notification)--->|
+    |                                              |
+
+```
+## Data type
+
+ - In this section explains about data type that is defined for Hatohol. They use date type that is defined in the JSON-RPC internally.
+
+|Name|JSON type|description|
+|:---|:---------|:---|
+|TimeStamp|string|This type stores times of day. Use UTC. Format is YYYYMMDDhhmmss.nnnnnnnnn. YYYY, MM, DD, hh, mm, ss, and nnnnnnnnn each express the A.D., Month, Day, hour, minute, second, nano second. Can abbreviate nano second. In case of the above or nano second digits less than nine, insert zero to spare parts(Ex.100 -> 100.000000000, 100.1234 -> 100.123400000).|
+|Boolean|true, false|This type stores boolean. Assign true or false to show authenticity of value.|
+|Number|number|This type stores number type value. Need to keep the number within the 0 and 2147483647.|
+|String255|string|This type stores string type value. Need to keep the number of characters within less than 255 characters.|
```

Comment:
This type is used to store string value. It needs to be the number of characters less than 255 characters.